src/TransportFactory.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace DataDog;
+
+interface TransportFactory
```

Comment:
Could you add comments to each of these new files describing the intention/use case for these new interfaces?

Reply:
I've added phpdoc comments.